doto-backend/index.js

```diff
@@ -26,7 +26,7 @@ db.once('open', function(){
     console.log("Connected to MongoDB.");
 });
 db.on('error', function(){
-    console.log(err);
+    
 });
 
 app.use(express.urlencoded({ extended: true }))
```

Comment:
can you remove this and change the /api-docs to /

Reply:
@fsan110 Yea this was causing issues with builds.